tests/test_modeling_tf_rag.py

```diff
@@ -861,7 +861,7 @@ def test_rag_token_generate_batch(self):
             " the 1970s",
             " 7.1. 2",
             " 13",
-            " step by step",
+            " evolution",
```

Comment:
@ratthachat for some reason one single line was different for the TF version. If you feel very motivated you could try to see what's going on there - it might also just be the case that there are those small differences between TF and PT